templates/ArtworkList.php

```diff
@@ -0,0 +1,20 @@
+<?
+
+$artworks = $artworks ?? [];
+?>
+<ol class="artwork-list list">
+<? foreach($artworks as $artwork){ ?>
+	<li>
+		<div class="thumbnail-container">
+			<picture>
+				<a href="/artworks/<?= $artwork->Slug ?>"><img src="<?= $artwork->ThumbUrl ?>" property="schema:image"/></a>
```

Comment:
nit: Strictly speaking a picture element "contains zero or more `<source>` elements and one `<img>` element" (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture).

You could wrap the `a` around the entire `picture` element to make it valid HTML (even though all browsers will display it fine as is). The same thing is present in `ArtworkDetail.php` (which I didn't catch in the previous review, sorry).

Reply:
Yes, you're totally right. I made that change, which then also required some CSS changes. Those CSS changes sparked some other ideas to improve the `ArtworkList` style, so I think it's even better now.